packages/thirdweb/src/wallets/smart/lib/userop.ts

```diff
@@ -54,6 +54,7 @@ export async function waitForUserOpReceipt(
     userOpHash: Hex;
     timeoutMs?: number;
     intervalMs?: number;
+    skipDecodeRevertReason?: boolean;
```

Comment:
Is this used at all here?

Reply:
actually, now since we are not using createUnsignedOp() & signUserOp() we might not need this.